assets/css/main.scss

```diff
@@ -212,6 +212,39 @@ hr.separator {
 
 // ==== PEOPLE PICKER ==== //
 
+.people-grid {
+  display: grid;
+  grid-auto-rows: 1fr;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+.advisor-grid-tile {
```

Comment:
Does the following class need to be nested in `.advisor-grid-tile`?

Reply:
I _think_ not... it should be specific enough, right?

Reply:
Yep!